packages/create-bison-app/scripts/createDevAppAndStartServer.js

```diff
@@ -112,6 +109,4 @@ if (require.main === module) {
   init();
 }
 
-module.exports = {
-  BISON_DEV_APP_VARIABLES_FILE,
-};
+module.exports = {};
```

Comment:
Is this needed? It looks like you're exporting nothing from this module.

Reply:
Removed